functional-tests/resources/redact_and_replace_payload.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
We really have to be consistent with the file vs inline payloads. 
Anyway, this will now fit the new infra for integration tests inside of anonymizer (as it serves a package as well) 
If you haven't changes anything in app.py, functional tests shouldn't be changes as well, 
I would postpone pushing this scenario until [this](https://github.com/microsoft/presidio/pull/455) will be merged

Reply:
I think we should have this scenario and change it in the future with user story.
I don't want to be dependent on another PR.
It takes too long for now.

Reply:
Now its in anyway

Reply:
I am sure I will. :)